periodic/periodic.go

```diff
@@ -575,3 +579,34 @@ func (r *RunnerResults) ID() string {
 	}
 	return base
 }
+
+//TODO remove above if condition when receive and sent byte count is added to GRPC as well.
```

Comment:
I think you mean below, not above

Reply:
you don’t want to update the comment ?

Reply:
Actually, I changed it but there was a problem with todo. Lint was asking for a empty line between method comment and todo. I did revert. After that I forgot to update it back. Sorry.